### ISSUE TITLE
OnionBrowser not accessible with Voiceover on IOS for visually impaired users

### ISSUE THREAD

**Issue by markada5** [OnionBrowser/OnionBrowser#297]
OnionBrowser is not accessible for visually impaired users on IOs.
On some websites, it's impossible to navigate with Voiceover because the browser refreshes the page in a way that Voiceover oses focus.

In addition, it's not possible to use the standard Voiceover gestures to activate different functions in OnionBrowser.

**Comment by tladesignz** [comment]
True. Thank you for pushing our noses onto this. 

I'm not sure we can do anything with this straight away. Our funding currently is very specific towards other (unfortunately more important goals), but I'll keep that in mind and I will push for it to resolve this as soon as we fix our biggest problem #231, without which all others are a futile endeavor.

@m4mb01t4l14n0, @mtigas: Let us please make this an important goal for when we resolved `WKWebView` support!
